CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,57 @@
+# Code of Conduct - Freedom of the Press Foundation
+
+Like the the broader technical and activism communities as a whole, the Freedom of the Press Foundation (FPF) team is made up of a mixture of staff and volunteers from all over the world, working on every aspect of our mission—including operations, software development, mentorship, and building connections with great people and organizations.
+
+To work together effectively in a large, diverse and open community, we have a few ground rules that we expect everyone to adhere to, be it FPF staff and board members, volunteers and event attendees; mentors, veterans, novices, or those seeking help and guidance.
+
+This isn’t an exhaustive list of things that you cannot do. Rather, take it in the spirit in which it’s intended—a guide to make it easier to enrich all of us and the communities in which we participate.
+
+This code of conduct applies to all spaces and events managed by the Freedom of the Press Foundation. This includes physical offices, GitHub repositories, online chat systems, the Support Forum, FPF hosted or sponsored events, and any other forums created by the Foundation which our team uses for communication. In addition, we take violations of this code outside these spaces into account when determining a person's ability to participate within them.
+
+If you believe someone is violating the code of conduct, we ask that you report it by talking to Emmanuel, or emailing **emmanuel@freedom.press**. If, for whatever reason, you are uncomfortable reporting the issue to Emmanuel, you may contact FPF’s Executive Director (**trevor@freedom.press**).  
```

Comment:
I think this section should come at the end. There should be a section saying "do and don't do this" followed by a "what to do if things are bad."

Second, there should be a single `foo@freedom.press` for this and we should specify who that is forwarded to. We should also list people's key fingerprints here.

There should be a statement saying something like "If you are uncomfortable emailing the entire list or someone on the list is the problem, you may email people individually."

Reply:
My knowledge about CoC says that we should mention individual names and email address and gpg keys. Most folks will not like to send to a list address which can be received by many folks.

Reply:
Then you do both. This seems fairly commons. Have a list and then enumerate everyone on the list and give their emails/keys.

Reply:
Agreed on @kushaldas's point re emailing specific folk. There are many instances that someone might be comfortable only sharing issues with one or two people, even if they might feel that everyone in a list would take it seriously.

I'm a bit skeptical about a list. That would require more consideration. Who maintains this list? What happens if no one responds from the list?

Reply:
A list is a permanent email address for handling this that makes it seem more permanent. How its handling is outside the scope of the CoC. Also, who would handle an email if it was send to you, @trevortimm, and @redshiftzero? One of the three of you has to respond, and a list is no different. But also this suggestion isn't a blocker on acceptance. Just it seems like what many other CoC's do, so maybe we should follow suit.